src/app/app.component.ts

```diff
@@ -1,69 +1,71 @@
 import { Component, EventEmitter, Output, QueryList, ViewChildren } from '@angular/core';
-import { TodoList } from './types/todo'
+import { TodoList } from './types/todo';
 import { ToDoListService } from './services/to-do-list.service';
+import { TaskData } from './types/task';
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.scss'],
 })
-
 export class AppComponent {
+    constructor(public toDoListService: ToDoListService = new ToDoListService()) {}
```

Comment:
```suggestion
    constructor(public toDoListService: ToDoListService = ToDoListService) {}
```
зачем создавать новый экземпляр? 
теряется вся суть использования сервиса